src/components/snapshots/useSnapshot.ts

```diff
@@ -87,7 +87,7 @@ const useSnapshot: SnapshotOperation = async ({
     // Import localisations
     if (options?.includeLocalisation) {
       try {
-        execSync(`cp -r  '${snapshotFolder}/localisation/' '${LOCALISATION_FOLDER}/' `)
+        execSync(`cp -r  '${snapshotFolder}/localisation/.' '${LOCALISATION_FOLDER}' `)
```

Comment:
This was the bugger that stumped me for ages. In develop the localisation files were being restored just fine, but on the server they were ending up nested a level deep. Subtle differences in the behaviour of different versions of `cp` I guess 🤷‍♂️

Reply:
I noticed this on my local copy while loading new snapshots! Good fix : )